sections/algorithmic_components.tex

```diff
@@ -544,36 +544,43 @@ \subsubsection{Single Visit}
 \end{itemize}
 
 
-
 \subsubsection{Joint Multi-Visit}
 \label{sec:acJointPhotometricFit}
 
-For DRP, all the observations can be combined in the so-called ubercal procedure to generate the best possible measurement of the relative flux of each source.
+In joint photometric calibration, all observations within a tract are combined to jointly estimate the best possible measurement of the relative flux of each source and a (gray) photometric scaling for each visit.
 
+This includes:
 \begin{itemize}
-\item Derive SEDs for calibration stars from colors and reference catalog classifications.
-\item Utilize additional information from wavelength dependent photometric calibration built by calibration products production to convert observed flux to a flux through a standard atmosphere.
-\item Fit zeropoint and possibly perturbations to all CCDs on multiple visits simultaneously after matching to reference catalog.
+\item Deriving SEDs for calibration stars from colors and reference catalog classifications.
+\item Utilizing additional information from wavelength-dependent photometric calibration built by calibration products production to convert observed flux to a flux through a standard atmosphere.
```

Comment:
I don't think that we know that the monitor telescope will provide everything we need.  E.g. is the y-band extinction through cloud always gray (no OH absorption) -- if not, we need to use stellar colours.

It is possible that we can separate the LSST data into "photometric" and "other" nights, and use only the photometric nights as inputs into übercal,  but I don't think I know this for sure.  As such, the \"ubercal code needs to be prepared to solve for some atmospheric effects (as does e.g. the DES calibration code).